config/initializers/devise_remote_user.rb

```diff
@@ -2,7 +2,7 @@
 
 DeviseRemoteUser.configure do |config|
   config.env_key = lambda do |env|
-    if Rails.env.development? && ENV['REMOTE_USER']
+    if Rails.env.development? && ENV.fetch('REMOTE_USER', nil)
```

Comment:
Can you say more about this change as well?

Reply:
Should be the same as before (I poked around in the Rails console).  This and the previous ENV change were made by a `rubocop -a` after adding the new configs it showed on standard out when running rubocop.

Reply:
Ah, ok. Thanks, rubocop! Merging.